Source/SLSC Switch/Docs/slscswitch-errors.txt

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<nidocument>
+<nicomment>
+https://github.com/ni/niveristand-routing-and-faulting-custom-device/
+</nicomment>
+<nierror code="-732050">
+Invalid endpoint configuration type.
+</nierror>
+<nierror code="-732051">
+The specified endpoint does not exist.
+</nierror>
+<nierror code="-732052">
+The reference ID provided does not match the expected name or cannot be found.
```

Comment:
@buckd note that some of these are duplicated errors (in message and semantics, but not in number) from R+F.

I'm torn between having each custom device vendor its own error messages exclusively, vs. re-using error codes provided by R+F.

Reply:
I am also torn, especially since we are using VeriStand error codes in some places. I'm fine with duplicating as I don't think we're going to run out of allocated codes.